Editor/Helpers/CinemachineDefaultMouseInput.cs

```diff
@@ -0,0 +1,92 @@
+#if CINEMACHINE_UNITY_INPUTSYSTEM
+using Cinemachine.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cinemachine
+{
+    /// <summary>
+    /// Provides a simple API to create a button that can add a CinemachineInputProvider component with a default
+    /// input asset controlling XY axis from the InputSystem package to the gameObject if it does not
+    /// already have CinemachineInputProvider.
+    /// </summary>
+    class CinemachineDefaultMouseInput
+    {
+        static CinemachineDefaultMouseInput s_Instance;
+        InputActionReference m_InputActionReference = null;
+        
+        /// <summary>
+        /// Initialize-on-demand singleton.
+        /// </summary>
+        /// <returns>Initialized instance</returns>
+        public static CinemachineDefaultMouseInput GetInstance() {
+            if (s_Instance == null) {
+                s_Instance = new CinemachineDefaultMouseInput();
+            }
+            return s_Instance;
+        }
+
+        CinemachineDefaultMouseInput()
+        {
+            var inputActionAsset = AssetDatabase.LoadAssetAtPath<InputActionAsset>("Packages/com.unity.inputsystem/" +
+                "InputSystem/Plugins/PlayerInput/DefaultInputActions.inputactions");
+
+            using var inputActions = inputActionAsset.GetEnumerator();
+            var hasNext = true;
+            while (hasNext)
+            {
+                if (inputActions.Current != null &&
+                    inputActions.Current.ToString() == "Player/Look[/Mouse/delta,/Pen/delta]")
+                {
+                    var look = inputActions.Current;
+                    m_InputActionReference = InputActionReference.Create(look);
+                    m_InputActionReference.name = "Generic Look";
```

Comment:
Why call it "Generic Look"?  It's confusing because the user won't know where it comes from.  Would it make sense to call it "Player/Look", same as what you searched for?